utility/module_test.go

```diff
@@ -0,0 +1,123 @@
+package utility
+
+import (
+	"log"
+	"os"
+	"testing"
+
+	"github.com/pokt-network/pocket/persistence"
+	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/runtime/configs"
+	"github.com/pokt-network/pocket/runtime/test_artifacts"
+	"github.com/pokt-network/pocket/runtime/test_artifacts/keygen"
+	"github.com/pokt-network/pocket/shared/messaging"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	dbURL string
+)
+
+func TestMain(m *testing.M) {
```

Comment:
I love how this `TestMain` naturally fits in with the other functions! :raised_hands: 

nit: I found the name `module_test.go` a bit misleading, I was expecting to see a test at some point but had to infer what's going on here. Wdyt about renaming it to something like `utils_test.go`? Perhaps we should establish a convention for this; wdyt about adding a `TECHDEBT` or `DISCUSS` comment about that as well?

Reply:
1. Renamed to `utility/main_test.go`
2. #709
3. Added `// NB: `TestMain` serves all tests in the immediate `utility` package and not its children`

Reference: https://github.com/pokt-network/pocket/pull/683/commits/e26608aabfb21553939e3bf2ed57fc5023b52755